openapi-cli/src/main/java/io/ballerina/openapi/generators/client/BallerinaAuthConfigGenerator.java

```diff
@@ -268,6 +279,32 @@ public TypeDefinitionNode getConfigRecord(OpenAPI openAPI) throws BallerinaOpenA
 
     }
 
+    /**
+     * Generate `AuthConfig` record when combination of ApiKey and HTTPOrOAuth authentication is supported.
+     *
+     * <pre>
+     *     # Provides Auth configurations needed when communicating with a remote HTTP endpoint.
+     *     public type AuthConfig record {|
+     *         # Auth Configuration
+     *         http:BearerTokenConfig|http:OAuth2RefreshTokenGrantConfig|ApiKeysConfig auth;
+     *     |};
+     * </pre>
+     * @return {@link TypeDefinitionNode}   Syntax tree node of config record
```

Comment:
Please add extra line after the function description. 
```suggestion
     * </pre>
     *
     * @return {@link TypeDefinitionNode}   Syntax tree node of config record
```